File: app/src/main/java/com/openclassrooms/realestatemanager/service/ApiInterface.java
package com.openclassrooms.realestatemanager.service;



import static com.openclassrooms.realestatemanager.BuildConfig.MAPS_API_KEY;

import com.openclassrooms.realestatemanager.models.resultGeocoding.ResultGeocoding;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ApiInterface {
    @GET("geocode/json?&key=" + MAPS_API_KEY)
    Call<ResultGeocoding> getResultGeocodingResponse(@Query("address") String address);
}

File: app/src/main/java/com/openclassrooms/realestatemanager/service/ApiInterface.kt
package com.openclassrooms.realestatemanager.service

import com.openclassrooms.realestatemanager.BuildConfig
import retrofit2.http.GET
import com.openclassrooms.realestatemanager.domain.models.resultGeocoding.ResultGeocoding
import retrofit2.Response
import retrofit2.http.Query

interface ApiInterface {
    @GET("geocode/json?&key=" + BuildConfig.MAPS_API_KEY)
    suspend fun getResultGeocodingResponse(@Query("address") address: String?): Response<ResultGeocoding>
}